test/unit/controllers/movements/GoodsDepartedControllerSpec.scala

```diff
@@ -36,13 +36,15 @@ class GoodsDepartedControllerSpec extends ControllerLayerSpec with MockCache {
 
   private val page = mock[goods_departed]
 
-  private def controller(answers: Answers = DepartureAnswers()) =
+  private val consignmentReferences = ConsignmentReferences(ConsignmentReferenceType.D, "referenceValue")
+
+  private def controller(answers: Answers = DepartureAnswers(consignmentReferences = Some(consignmentReferences))) =
     new GoodsDepartedController(SuccessfulAuth(), ValidJourney(answers), cacheRepository, stubMessagesControllerComponents(), page)(global)
 
   override def beforeEach(): Unit = {
     super.beforeEach()
 
-    when(page.apply(any())(any(), any())).thenReturn(HtmlFormat.empty)
+    when(page.apply(any(), anyString())(any(), any())).thenReturn(HtmlFormat.empty)
```

Comment:
Not sure if using `anyString` make any value. If the field is as String type, any is enough because in general you're not able to put anything else.

Always to avoid more imports we can go with `any[String]`.

But this is correct